lisa/tools/reboot.py

```diff
@@ -149,7 +149,7 @@ def command(self) -> str:
     def _check_exists(self) -> bool:
         return True
 
-    def reboot(self, time_out: int = 600) -> None:
+    def reboot(self, time_out: int = 1200) -> None:
```

Comment:
It doesn't need to change the default value. Specify the value when it's called.